### PR TITLE
Colonypick

### PR DESCRIPTION
**Pull request by jarpinosynthace**

### CODE REVIEW

**Comment by jarpinosynthace** [comment]
@synthacebot test this please

**Comment by gogogadgetbioprocess** [comment]
@synthacebot test this please

**Comment by jarpinosynthace** [comment]
@synthacebot test this please

File: microArch/driver/liquidhandling/makelhpolicy.go [hunk 1]
```diff
@@ -423,11 +423,12 @@ func MakeColonyPolicy() LHPolicy {
 	policy["ASPSPEED"] = 3.0
 	policy["DSPSPEED"] = 3.0
 	policy["ASP_WAIT"] = 1.0
-	policy["POST_MIX"] = 3
+	policy["POST_MIX"] = 1
```
Comment (m-i-sadowski) [inline microArch/driver/liquidhandling/makelhpolicy.go]: Please make a new liquid policy and make changes to it... prevents potentially conflicting with other uses of same policy. You can inherit from this one.

File: microArch/factory/make_plate_library.go [hunk 2]
```diff
@@ -428,6 +385,30 @@ func makePlateLibrary() map[string]*wtype.LHPlate {
 	plate = wtype.NewLHPlate("reservoir", "unknown", 1, 1, 45, "mm", welltypereservoir, 58, 13, 0, 0, 10)
 	plates[plate.Type] = plate
 
+	// Onewell SBS format Agarplate with colonies on shallowriser (50ml agar) very high res
```
Comment (m-i-sadowski) [inline microArch/factory/make_plate_library.go]: Please move new labware definitions to dedicated functions

**Comment by gogogadgetbioprocess** [comment]
I've just added a colonymix policy which contains the original policy; this is consistent with the style of other policies (e.g. dna, dnamix) and leaves the fast option as the default.

**Comment by gogogadgetbioprocess** [comment]
@synthacebot test this please

**Comment by gogogadgetbioprocess** [comment]
@synthacebot test this please

**Pull request merged**